apps/site/lib/site_web/views/teaser_view.ex

```diff
@@ -1,16 +1,22 @@
 defmodule SiteWeb.CMS.TeaserView do
   @moduledoc """
-  Handles rendering of teaser partials from the CMS.
+  Handles display elements of teaser partials from the CMS.
   """
   use SiteWeb, :view
 
-  import SiteWeb.CMSHelpers, only: [cms_route_to_class: 1]
+  import SiteWeb.CMSHelpers
```

Comment:
Can we keep the `only` and expand it to include each of the functions we're using, so the source of functions encountered in the code is easily deducible?